mod/Gfluct3.mod

```diff
@@ -232,15 +232,16 @@ VERBATIM
 ENDVERBATIM
 }
 
-:BEFORE BREAKPOINT {
-PROCEDURE foo() {
+BEFORE BREAKPOINT {
```

Comment:
@nrnhines : Just curious - you changed this PROCEDURE to BEFORE BREAKPOINT block for any technical reason?

I see that `BEFORE/AFTER` is not implemented in new nmodl. I will look into implementing this.